traffic_ops/experimental/ui/app/src/styles/main.scss

```diff
@@ -140,3 +139,10 @@ body.nav-sm .container.body .main-content {
   }
 
 }
+
+@media(min-width:992px) {
```

Comment:
yes, that's how media queries are written. not sure why GH wants to highlight it..